### PR TITLE
wallet page tweaks

### PR DESCRIPTION
**Pull request by johnkuney**
specifying operating systems for the desktop wallets and making the sidebar anchor link to corresponding wallet sections

### CODE REVIEW

File: v2/src/components/sidebar/sidebar.js [hunk 1]
```diff
@@ -2,14 +2,28 @@ import React from "react"
...
     <Col md={4}>
       <div className={Sidebarstyle.card}>
         <h3>{title}</h3>
-        {content.map((item, i) => {
-          return (
-            <Row key={i} className={Sidebarstyle.cardIconRow}>
+        {link &&
+          content.map(item => (
+            <Row className={Sidebarstyle.cardIconRow}>
+              <a href={item.link}>
```
Comment (deadalnix) [inline v2/src/components/sidebar/sidebar.js:13]: This seem to be the only difference between the two. You can make the function in the map a bit simpler instead.

```
content.map(item => {
    let tile = (<>
                <Col xs={3} className={Sidebarstyle.cardIcon}>
                  {item.icon}
                </Col>
                <Col xs={9} className={Sidebarstyle.cardIconText}>
                  <h4>{item.title}</h4>
                  <p>{item.paragraph}</p>
                </Col>
    </>)

    if (item.link) {
      tile = <a href={item.link}>{tile}</a>
    }

    return             <Row className={Sidebarstyle.cardIconRow}>{tile}</Row>
  }
}
```

File: v2/src/pages/wallets.js [hunk 2]
```diff
@@ -110,6 +116,7 @@ const WalletsPage = () => {
               "What kind of wallet should I use?",
               "sidebar title 'wallets' page"
             )}
+            link={true}
```
Comment (deadalnix) [inline v2/src/pages/wallets.js:119]: This is not necessary.

File: v2/src/pages/wallets.js [hunk 3]
```diff
@@ -125,7 +132,7 @@ const WalletsPage = () => {
         <FeaturedTiles tiles={featuredWallets} md={6} />
       </Container>
       <Container>
-        <h2 className="centerh2">
+        <h2 className="centerh2" id={"mobile-wallets"}>
```
Comment (deadalnix) [inline v2/src/pages/wallets.js:135]: You don't need curly braces.

File: v2/src/pages/wallets.js [hunk 4]
```diff
@@ -140,33 +147,33 @@ const WalletsPage = () => {
         </Tabs>
       </Container>
       <Container>
-        <h2 className="centerh2">
+        <h2 className="centerh2" id={"desktop-wallets"}>
```
Comment (deadalnix) [inline v2/src/pages/wallets.js:150]: dito

**Review by deadalnix — CHANGES_REQUESTED** [review]

**Comment by deadalnix** [comment]
In general, you want to split your patches so that they do one thing. This does two (anchor links and proper OS support for desktop wallets) meaning both will have to be good before any land.

File: v2/src/components/sidebar/sidebar.js (hunk 1 above)
Comment (johnkuney) [reply v2/src/components/sidebar/sidebar.js]: Ah nice, yeah thats a cool way to do it. Updated

File: v2/src/pages/wallets.js (hunk 2 above)
Comment (johnkuney) [reply v2/src/pages/wallets.js]: removed

File: v2/src/pages/wallets.js (hunk 3 above)
Comment (johnkuney) [reply v2/src/pages/wallets.js]: fixed

File: v2/src/pages/wallets.js (hunk 4 above)
Comment (johnkuney) [reply v2/src/pages/wallets.js]: fixed

File: v2/src/components/sidebar/sidebar.js [hunk 5]
```diff
@@ -2,23 +2,27 @@ import React from "react"
 import { Row, Col } from "react-bootstrap"
 import Sidebarstyle from "./sidebar.module.css"
 
-const Sidebar = ({ content, title }) => {
+const Sidebar = ({ content, title, link }) => {
```
Comment (deadalnix) [inline v2/src/components/sidebar/sidebar.js:5]: link is not used anymore

**Review by deadalnix — CHANGES_REQUESTED** [review]
You missed a detail, but overall, LGTM

File: v2/src/components/sidebar/sidebar.js (hunk 5 above)
Comment (johnkuney) [reply v2/src/components/sidebar/sidebar.js]: grrr good catch. Fixed

**Pull request merged**